app/models/message.rb

```diff
@@ -27,4 +28,20 @@ def format_phone_number
 
     self.phone = self.phone.gsub(/[^0-9]/, '')
   end
+
+  RELEVANT_TERMS = %w[
+    delegate
+    bernie
+    sanders
+    hillary
+    clinton
+    hrc
+  ]
+
+  def contents_is_relevant
+    unless RELEVANT_TERMS.any? { |term| contents.include?(term) }
+      errors.add :contents,
+        "It looks like the contents of your message isn't relevant..."
```

Comment:
Is there a better error message that lets them know exactly how to fix the error state? Otherwise this error will be very frustrating.